### DISCUSSION TITLE
Outline public IP not working on ECS

### DISCUSSION THREAD

**Discussion by siddharthakhuntia-lohum** [outline/outline#6657] [Self Hosting]
I have deployed outline on AWS ECS Fargate with OIDC auth.
I dont have domain name to give it to it. But have enabled public IP on ECS. I am unable to open the public IP even though i have allowed all incoming requests in the assigned security group. 
![image](https://github.com/outline/outline/assets/157572803/c02ca893-3d0a-437d-bab9-500e7b98cde2)
![image](https://github.com/outline/outline/assets/157572803/84b8627b-f098-4830-980d-e21aec7fd164)

**Comment by tommoor** [comment]
A valid domain name is a requirement of the software working correctly.

It's been an assumption since the beginning so I'm not sure what changes are needed for it to work with only an IP address.
